### ISSUE TITLE
Provide SPARQL endpoint

### ISSUE THREAD

**Issue by jiskattema** [research-software-directory/RSD-as-a-service#679]
To publish all our software, projects, and output in a FAIR way, it would be nice to have a SPARQL endpoint.
This would also allow (easier) linking of the RSD to other data sources and websites.

The use of SPARQL endpoints is also common practice in the humanities domains -- and probably the life sciences.

**Comment by jmaassen** [comment]
Yes, SPARQL can provide a nice API to the data. We'll put it on the list of APIs we'll have a closer look at.

**Comment by ewan-escience** [comment]
We don't plan to do this, as there seems to be no demand for it, is seems complicated to implement, and we have [other metadata endpoints](https://research-software-directory.org/documentation/API/) available.
